README.md

```diff
@@ -91,7 +106,12 @@ Define a function called `doSomething`. It should take one argument, called
 
 Your Answer:
 ```js
-// write code here
+function doSomething(thingToDo){
+    function thingToDo (){    
+    }
+}
```

Comment:
Check the [solution](https://github.com/ga-wdi-quizzes/week-04/tree/solution#question-7).